content/docs/command-reference/add.md

```diff
@@ -148,15 +165,29 @@ not.
   > Note that external outputs typically require an external cache setup. See
   > link above for more details.
 
-- `--desc <text>` - user description of the data (optional). This doesn't affect
+- `--to-remote` - transfer data straight to remote, when the used system doesn't
```

Comment:
same here, a bit too much with the details. we can be brief here, and put a backlink to the section above since we already have it.

I would start with something a bit more clear in a sense that we still use `dvc add` terminology. 

`Add target data into DVC and create a .dvc file, but instead of caching it into DVC cache, transfer it straight to remote storage ... Check this section for the details . If this option is specified target can be any cloud or local URL, not necessarily a local file or directory from the workspace as it is required in the regular `dvc add` workflow. `